crawler4j-commons/src/main/java/edu/uci/ics/crawler4j/crawler/DynamicCrawlConfig.java

```diff
@@ -0,0 +1,143 @@
+/*-
+ * #%L
+ * de.hs-heilbronn.mi:crawler4j-core
+ * %%
+ * Copyright (C) 2010 - 2021 crawler4j-fork (pre-fork: Yasser Ganjisaffar)
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package edu.uci.ics.crawler4j.crawler;
+
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Extends {@link CrawlConfig} to add configuration properties for the Selenium's <em>WebDriver</em>.
+ */
+public class DynamicCrawlConfig extends CrawlConfig {
+
+    public enum WebDriverType {
+        chrome,
+        firefox
+    }
+
+    private WebDriverType webDriverType = WebDriverType.firefox;
+
+    /**
+     * If dynamic content crawling is used, the maximum time to wait for dynamic content loading in seconds.
+     */
+    private int maxWaitForDynamicContentInSeconds = 2;
+
+    /**
+     * Absolute path to the driver binary on the local filesystem.
+     */
+    private Path webDriverPath;
```

Comment:
AFAIK it's not needed nowadays: https://www.selenium.dev/documentation/selenium_manager/#automated-driver-management